core/src/main/java/org/elasticsearch/ingest/ConfigurationUtils.java

```diff
@@ -251,6 +250,28 @@ public static ElasticsearchParseException newConfigurationException(String proce
         return processors;
     }
 
+    public static TemplateService.Template compileTemplate(String processorType, String processorTag, String propertyName,
+                                                           String propertyValue, TemplateService templateService) {
+        try {
+            return templateService.compile(propertyValue);
+        } catch (Exception e) {
```

Comment:
I see, I guess I am not familiar with such exceptions. If all such exceptions extend `ElasticsearchException` then I suppose there is no real need to wrap the exception, but instead just add the appropriate ingest headers